domain/notifier.go

```diff
@@ -28,4 +28,7 @@ type NotificationMessage struct {
 type Notification struct {
 	User    string
 	Message NotificationMessage
+
+	AppealID string
+	GrantID  string
```

Comment:
```suggestion
	Labels map[string]string
```
How about having this generic one? and the notifier client will log all key-values within `Labels`

Reply:
makes sense. Will update.